bitnami/kafka/Chart.yaml

```
@@ -29,4 +29,4 @@ name: kafka
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/kafka
   - https://kafka.apache.org/
-version: 20.0.6
+version: 20.0.7
```

As you are adding new features, please increase the minor version. https://semver.org/
```suggestion
version: 20.1.0
```